.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,5 @@
 
 **JIRA:** Link to JIRA ticket
 
-**Merge checklist:**
-- [ ] All reviewers approved
-- [ ] CI build is green
```

Comment:
Agreed this isn't necessary *if the repo* is configured to require green before merge, which this one does. So, this is ok to drop.